sos_brazil/exceptions.py

```diff
@@ -38,3 +38,8 @@ def __init__(
 
         self.detail = f"{key.title()} cannot be updated."
         self.status_code = status_code
+
+
+class InvalidCnpjException(APIException):
```

Comment:
Since we will have an `InvalidDateFormat` too, this could be a `InvalidFormatException` and my suggestion for the message would be something like this:

```python
{
  "error": "Invalid CNPJ format.",
  "expected_format": "XX.XXX.XXX/XXXX-XX"
}
```

What do you think?

Reply:
Anyways, i can created that exception later if you want, just create a task and assign me.

Reply:
I managed to do it as you suggested.